src/components/OrganizationAttributes.js

```diff
@@ -14,28 +14,28 @@ import Tag from "./Tag"
 
 export const OrganizationCategory = ({ text, ...props }) => (
   <Tag {...props}>
-    <FontAwesomeIcon icon={faBox} className="mr-1" />
+    <FontAwesomeIcon icon={faBox} className="mr-2" />
```

Comment:
All the components on this page are used on the OrganizationFilter / OrganizationCard. 
So this modification will affect those components. Is it what we want?

Reply:
i separated the organisation template from the organisation attributes components so it won't be dependent on each other, as it made ui problems in the organisation template.
this styling changes made felt more reasonable on the organisations page domain.